MoltenVK/MoltenVK/GPUObjects/MVKDevice.mm

```diff
@@ -3077,11 +3077,24 @@ static uint32_t mvkGetEntryProperty(io_registry_entry_t entry, CFStringRef prope
 		pWritableExtns->vk_KHR_fragment_shader_barycentric.enabled = false;
 		pWritableExtns->vk_NV_fragment_shader_barycentric.enabled = false;
 	}
-	// gpuAddress requires Tier2 argument buffer support (per feedback from Apple engineers).
-	if (_metalFeatures.argumentBuffersTier < MTLArgumentBuffersTier2) {
+    
+    // The relevant functions are not available if not built with Xcode 14.
+#if !MVK_XCODE_14
```

Comment:
This will read a bit better if you swap the order of the if/else clauses here, so it reads `#if MVK_XCODE_14` (no `!`), and put the two fallback lines in the `#else` section.